src/pages/resources/recipes/persistent-storage-migration/index.md

```diff
@@ -0,0 +1,50 @@
+---
+keywords:
+  - Creative Cloud
+  - API Documentation
+  - UXP
+  - Plugins
+  - JavaScript
+  - ExtendScript
+  - SDK
+  - C++
+  - Scripting
+title: Persistent Storage Migration in InDesign UXP
+description:
+---
+
+# Persistent Storage Migration
```

Comment:
Please structure this doc according to the recipe template https://github.com/AdobeDocs/uxp-indesign/blob/main/src/pages/resources/recipes/_recipe_template.md